accounts.go

```diff
@@ -18,7 +22,8 @@ func (lw *LibWallet) GetAccounts(requiredConfirmations int32) (string, error) {
 }
 
 func (lw *LibWallet) GetAccountsRaw(requiredConfirmations int32) (*Accounts, error) {
-	resp, err := lw.wallet.Accounts()
+	ctx := lw.shutdownContext()
+	resp, err := lw.wallet.Accounts(ctx)
```

Comment:
What I had in mind with introducing the `lw.shutdownContext()` was to use it directly in call sites such as this:
```
resp, err := lw.wallet.Accounts(lw.shutdownContext())
```